test/integration/globals/app-nav.js

```diff
@@ -46,18 +40,55 @@ context('App Nav', function() {
 
         expect(storageItem).to.be.false;
       });
-    cy
-      .getRadio(Radio => {
-        logoutStub = cy.stub();
-        Radio.reply('auth', 'logout', logoutStub);
-      });
 
     cy
       .get('.app-nav__header')
-      .should('contain', 'Cypress Clinic')
+      .should('contain', 'Workspace One')
       .should('contain', 'Clinician McTester')
       .as('mainNav');
 
+    cy
+      .get('@mainNav')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-workspace-name')
+      .should('contain', 'Cypress Clinic');
+
+    cy
+      .get('.picklist')
+      .find('.picklist__group')
+      .find('.picklist__item')
+      .first()
+      .should('contain', 'Workspace One')
+      .should('have.class', 'is-selected')
+      .next()
+      .should('contain', 'Workspace Two')
+      .should('not.have.class', 'is-selected');
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-bottom')
+      .contains('Help & Support')
+      .should('have.attr', 'href')
+      .and('contain', 'help.roundingwell.com');
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-bottom')
+      .contains('Sign Out')
+      .should('have.attr', 'href')
+      .and('contain', '/logout');
+
+    // NOTE: this closes the main nav droplist so it doesn't cover other nav links
+    cy
+      .get('.picklist')
+      .find('.picklist__group')
+      .find('.picklist__item')
+      .first()
+      .click();
```

Comment:
Needed to close the main nav droplist somehow because it was overlapping other elements in the nav and causing the test to fail.

This just clicks on the currently active workspace picklist item, which does nothing other than close the droplist.